### PR TITLE
keys: Don't allow slash in device-ca

### PR DESCRIPTION
**Pull request by doanac**
This command expects a filename and not a path. This change makes the error more clear.

fixes issue #344

### CODE REVIEW

File: subcommands/keys/ca_add_device_ca.go [hunk 1]
```diff
@@ -60,6 +60,12 @@ func doAddDeviceCa(cmd *cobra.Command, args []string) {
 		subcommands.DieNotNil(errors.New("At least one of --online-ca or --local-ca must be true"))
 	}
 
+	if len(localCaFilename) > 0 {
+		if strings.ContainsRune(localCaFilename, os.PathSeparator) {
+			subcommands.DieNotNil(errors.New("The --local-ca-filename argument must be filename and not a path"))
```
Comment (vkhoroz) [inline subcommands/keys/ca_add_device_ca.go:65]: I did not want to enforce this limitation, but I understand your justification.

Not sure if we have similar parameters in other ca subcommands.
Can you check?

**Review by vkhoroz — APPROVED** [review]
LGTM

File: subcommands/keys/ca_add_device_ca.go (hunk 1 above)
Comment (doanac) [reply subcommands/keys/ca_add_device_ca.go]: found another spot. address in force push with a helper function.

**Review by vkhoroz — APPROVED** [review]
LGTM

**Pull request merged**